tools/build.js

```diff
@@ -126,8 +129,38 @@ await Promise.all([
   cp('src/build/templates', join(OUT_DIR, 'build/templates'), { recursive: true, force: true }),
 ])
 
+async function ensureNoRegionalBlobsModuleDuplicates() {
+  const REGIONAL_BLOB_STORE_CONTENT_TO_FIND = 'fetchBeforeNextPatchedIt'
+
+  const filesToTest = await glob(join(OUT_DIR, '**/*.{js,cjs}'))
+  const unexpectedModulesContainingFetchBeforeNextPatchedIt = []
+  let foundInExpectedModule = false
+  for (const fileToTest of filesToTest) {
+    const content = await readFile(fileToTest, 'utf-8')
+    if (content.includes(REGIONAL_BLOB_STORE_CONTENT_TO_FIND)) {
+      if (fileToTest.endsWith('run/regional-blob-store.cjs')) {
+        foundInExpectedModule = true
+      } else {
+        unexpectedModulesContainingFetchBeforeNextPatchedIt.push(fileToTest)
+      }
+    }
+  }
+  if (!foundInExpectedModule) {
+    throw new Error(
+      'Expected to find fetchBeforeNextPatchedIt in run/regional-blob-store.cjs, but it was not found. This might indicate setup change that require bundling validation to be adjusted.',
+    )
+  }
+  if (unexpectedModulesContainingFetchBeforeNextPatchedIt.length !== 0) {
+    throw new Error(
+      `Bundling produced unexpected duplicates of 'regional-blob-store' module in following built modules:\n${unexpectedModulesContainingFetchBeforeNextPatchedIt.map((filePath) => ` - ${filePath}`).join('\n')}`,
+    )
+  }
```

Comment:
This produce error like this:
```
Error: Bundling produced unexpected duplicates of 'regional-blob-store' module in following built modules:
 - dist/esm-chunks/chunk-K4RDUZYO.js
 - dist/run/next.cjs
 - dist/run/handlers/cache.cjs
    at ensureNoRegionalBlobsModuleDuplicates (file:///Users/misiek/dev/pgs-next-runtime/tools/build.js:154:11)
    at async file:///Users/misiek/dev/pgs-next-runtime/tools/build.js:163:3
```
The main purpose is to prevent publishing mis-bundlend version of runtime from local machines